bidsify/bidsify.py

```diff
@@ -15,35 +13,34 @@
 # Local imports
 from bidsify.utils import run, manage_dicom_dir, maintain_bids
 
-
 def _get_parser():
     """
     Set up argument parser for scripts
     """
     parser = argparse.ArgumentParser(description='BIDS conversion and '
                                                  'anonymization.')
-    parser.add_argument('-d', '--dicomdir',
+    parser.add_argument('-d', '--dicomdir', type=Path,
                         required=True, dest='dicom_dir',
                         help='Directory or tar file containing raw data.')
-    parser.add_argument('-f', '--heuristics',
+    parser.add_argument('-f', '--heuristics', type=Path,
```

Comment:
We're ultimately going to want wiggle room on this for https://github.com/FIU-Neuro/cis-processing/issues/49.

Reply:
I still think that we should just default to Path for this, but I flipped the argument to optional and added a conditional so if we add a `reproin` flag we can just work around it

Reply:
Reproin won't be a separate flag though. It'll be an option for `--heuristics` (i.e., `--heuristics reproin`).